guides/contributing/Updating Changelogs.md

```diff
@@ -0,0 +1,65 @@
+# Updating Changelogs
+
+- [Changelog entries](#changelog-entries)
+- [Categories](#categories)
+- [Tools](#tools)
+
+A changelog is a file which contains a curated, chronologically ordered list of notable changes for each version of a package.
+The main purpose of changelogs (in general) is to inform users and contributors about changes that have been made in subsequent versions. However, in [`expo/expo`](https://github.com/expo/expo) repository, they have one more purpose — to help us decide how to bump the package version when we publish it (see [categories'](#categories) descriptions for more).
+
+Most of the packages inside [`packages`](https://github.com/expo/expo/tree/master/packages) directory have their own changelogs, that should get updated on every change we made in the package.
+
+## Changelog entries
+
+Each bullet point in the changelog is called an entry. It describes the change that has been introduced in the package. Here is what defines a good changelog entry:
+
+- It is descriptive and concise — explains well the change to a reader who has _zero context_ about the change. If you have trouble making it both concise and descriptive, err on the side of descriptive.
+- It is placed under `Unpublished` version and the appropriate category ([follow rules described here](#categories)).
+- It has links to PR's author and PR itself (`expo-ci` bot can suggest them for you, read more below in this section).
+- It may contain only text and links. Any other markdown elements (such as lists, headers, blockquotes, tables, images, inline HTML) are not permitted — if you need them to provide important tips for the users to adapt to the (breaking) change, you should put this in the PR's description or link to a separate document on [`expo/fyi`](https://github.com/expo/fyi).
```

Comment:
```suggestion
- It may contain only text and links. Any other markdown elements (such as lists, headers, blockquotes, tables, images, inline HTML) are not permitted — if you need them to provide important tips for the users to adapt to the (breaking) change, you should put this in the PR description or link to a separate document on [`expo/fyi`](https://github.com/expo/fyi).
```